docs/release-notes/8.1.asciidoc

```diff
@@ -1,6 +1,15 @@
 [[release-notes-header-8.1.0]]
 == 8.1
 
+[discrete]
+[[release-notes-8.1.2]]
+=== 8.1.2
+
+[discrete]
+[[bug-fixes-8.1.2]]
+==== Bug fixes and enhancements
+* The primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc.
```

Comment:
I didn't mention it in the issue, but this change only affects Windows. We do sign the macOS Endpoint as well (though it isn't named `elastic-endpoint.exe`). I don't feel strongly whether its worth calling out the OS here or leaving this as-is but am mentioning it in case you think it matters.